.markdownlint-cli2.yaml

```diff
@@ -1,6 +1,7 @@
 globs:
   - "concepts/**/*.md"
   - "exercises/concept/**/*.md"
+  - "exercises/practice/*/.docs/{hints.md,instructions.append.md}"
```

Comment:
We should add introduction.append.md here in case we make one of those in the future.